src/style.css

```diff
@@ -1,3 +1,79 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+header {
+  width: 100%;
+  height: 60px;
```

Comment:
no needs to hardcode height for elements with content inside. To make a correct height apply padding to a link